aten/src/ATen/native/cuda/jit_utils.cpp

```diff
@@ -811,14 +818,18 @@ c10::optional<std::string> get_cache_dir() {
     return {};
   }
 
-  // Cache path comes from PYTORCH_KERNEL_CACHE_PATH, then XDG_CACHE_HOME, then HOME environment variables
+  // Cache path comes from PYTORCH_KERNEL_CACHE_PATH, then TEMP or XDG_CACHE_HOME, then HOME environment variables
```

Comment:
can you please mention in this comment that `TEMP` is used only on windows (and `XDG_CACHE_HOME` on linux systems), and make a similar comment in the cuda note that describes caching behavior?